assembl/static2/js/app/utils/translations.js

```diff
@@ -903,7 +904,13 @@ const Translations = {
           headerDescription:
             "L'image doit avoir une hauteur de 450px de haut et une largeur de 1280px. Le poids ne doit pas dépasser 1Mo.",
           logoDescription:
-            "Le logo doit avoir une hauteur maximale de 78px et une largeur maximale de 200px. Le poids ne doit pas dépasser 1Mo. Le fond doit être transparent."
+            "Le logo doit avoir une hauteur maximale de 78px et une largeur maximale de 200px. Le poids ne doit pas dépasser 1Mo. Le fond doit être transparent.",
+          startDate: "Du",
+          endDate: "À",
```

Comment:
If we are talking about dates you would say "Du 12 Janvier **au** 15 Février" so you probably want "Au" here